distributed/scheduler.py

```diff
@@ -3933,7 +3935,7 @@ def update_graph(
             if tasks[k] is k:
                 del tasks[k]
 
-        cs: ClientState = parent._clients[client]
+        cs: ClientState = None  # parent._clients[client]
```

Comment:
We also make use of `cs` below

https://github.com/jakirkham/distributed/pull/3/files#diff-bbcf2e505bf2f9dd0dc25de4582115ee4ed4a6e80997affc7b22122912cc6591R4190

Reply:
Only if a task ended up in `touched_tasks`, which means we must have gone past the place where we set its value, no?